src/styles/global.scss

```diff
@@ -14,6 +14,7 @@
 
 .ws-heading {
   position: relative;
+  scroll-margin-top: 3rem;
```

Comment:
I'm not sure if I'm noticing any difference when removing this line, is it necessary for anything in particular?